src/prop/prop_engine.cpp

```diff
@@ -91,7 +91,7 @@ PropEngine::PropEngine(TheoryEngine* te,
   d_satSolver = SatSolverFactory::createDPLLMinisat(smtStatisticsRegistry());
 
   d_registrar = new theory::TheoryRegistrar(d_theoryEngine);
-  d_cnfStream = new CVC4::prop::TseitinCnfStream(
+  d_cnfStream = new CVC4::prop::CnfStream(
```

Comment:
Why this change (and the ones below?).  Was the derived class redundant?

Reply:
The CnfStream was only ever used as a TseitingCnfStream extension. So I collapsed the latter into the former.